src/classes/ADDR_Seasonal_SCHED.cls

```diff
@@ -35,7 +35,8 @@
 */ 
 public class ADDR_Seasonal_SCHED implements Database.Batchable<SObject>, Schedulable {
 
-    private static final Integer BATCH_SIZE = 10;
+    private static final Integer BATCH_SIZE = (UTIL_CustomSettingsFacade.getHouseholdsSettings().Seasonal_Addresses_Batch_Size__c != null) ?
```

Comment:
- If the batch size is not provided, negative or null, default should be 10. No need for `Math.abs()`.